### ISSUE TITLE
Any plan for multi-person identity-preserving with PULID?

### ISSUE THREAD

**Issue by Oo121oO** [logtd/ComfyUI-Fluxtapoz#64]
As title!
As it has been release since 2024/11/26.
Thanks a lot!

**Comment by logtd** [comment]
Do they already exist here https://github.com/balazik/ComfyUI-PuLID-Flux ?

**Comment by Oo121oO** [comment]
Nope! I saw it here: https://github.com/instantX-research/Regional-Prompting-FLUX?tab=readme-ov-file.

**Comment by APlisetskii** [comment]
Hi @logtd PuLID multi-person identity would be a great feature. I would be grateful if you could add support for such a workflow to your repository.
